pkg/controller/eventtype/reconcile_test.go

```diff
@@ -0,0 +1,216 @@
+/*
+Copyright 2018 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package eventtype
+
+import (
+	"fmt"
+	feedsv1alpha1 "github.com/knative/eventing/pkg/apis/feeds/v1alpha1"
+	controllertesting "github.com/knative/eventing/pkg/controller/testing"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/client-go/kubernetes/scheme"
+	"k8s.io/client-go/tools/record"
+	"testing"
+)
+
+func TestStub(t *testing.T) {
+	// TODO: Implement tests in this directory, please.
+	// This file is used to make code coverage on this directly count.
+}
+
+var (
+	// deletionTime is used when objects are marked as deleted. Rfc3339Copy()
+	// truncates to seconds to match the loss of precision during serialization.
+	deletionTime = metav1.Now().Rfc3339Copy()
+)
+
+const (
+	eventSourceName = "event-source"
+	etNamespace     = "test-namespace"
+	etName          = "test-event-type"
+	reconcileKey    = etNamespace + "/" + etName
+
+	fn1 = "fake-feed-name-1"
+	fn2 = "fake-feed-name-2"
+)
+
+func init() {
+	// Add types to scheme
+	feedsv1alpha1.AddToScheme(scheme.Scheme)
+}
+
+var testCases = []controllertesting.TestCase{
```

Comment:
normally this is defined inside the test that uses it. this is because you can have more than one test in this file or package and it is very clear which `testCases`  you are using.